Engine/Media/HardwareDecoder.cpp

```diff
@@ -252,7 +244,7 @@ AVCodec *MediaProcController::HardwareDecoderIFace::findDecoder(AVCodecContext *
 #if defined(IOS) || defined(MACOSX)
 	return HardwareDecoderVT::findDecoder(context);
 #elif defined(DROID)
-	return HardwareDecoderMC::findDecoder(context);
+	return const_cast<AVCodec *>(HardwareDecoderMC::findDecoder(context)); //this is very dirty and I do not like it one bit
```

Comment:
What's wrong with this? Why do you need a cast?

Reply:
I think it is code from when ffmpeg was 5.0

Reply:
I guess we need to cleanup it.